jenkins/promotion/promote-docker-ecr.jenkinsfile

```diff
@@ -0,0 +1,86 @@
+lib = library(identifier: 'jenkins@20211123', retriever: legacySCM(scm))
+
+pipeline {
+    options {
+        timeout(time: 1, unit: 'HOURS')
+    }
+    agent {
+        docker {
+            label 'Jenkins-Agent-al2-x64-c54xlarge-Docker-Host'
+            image 'opensearchstaging/ci-runner:ubuntu2004-x64-docker-buildx0.6.3-qemu5.0-awscli1.22-jdk11-v2'
+            args '-u root -v /var/run/docker.sock:/var/run/docker.sock'
+        }
+}
+    parameters {
+        string(
+            name: 'PROMOTE_PRODUCT',
+            description: 'Comma separated list of product with its image tag that we want to promote from staging to production. E.g.: opensearch:2.0.1.3910, opensearch-dashboards:2.0.1, data-prepper1.5.0-19.',
+            trim: true
+        )
+        string(
+            name: 'RELEASE_VERSION',
+            description: 'Official release version on production repository. E.g.: 2.0.1, 1.5.0.',
+            trim: true
+        )
+        booleanParam(
+            name: 'DOCKER_HUB_PROMOTE',
+            defaultValue: true,
+            description: 'Promote the staging images to Docker Hub with the tag choices.'
+        )
+        booleanParam(
+            name: 'ECR_PROMOTE',
+            defaultValue: true,
+            description: 'Promote the staging images to ECR with the tag choices.'
+        )
+        booleanParam(
+            name: 'TAG_LATEST',
+            defaultValue: true,
+            description: 'Tag the copied image as latest'
+        )
+        booleanParam(
+            name: 'TAG_MAJOR_VERSION',
+            defaultValue: true,
+            description: 'Tag the copied image to be on the latest image for any build within its major version. E.g.: 1.3.2 image will be tagged with 1 in the hub.'
+        )
+    }
```

Comment:
Please give proper naming to these params, they are confusing at best for anyone that is not familiar with the release processes.

Reply:
Updated the wording for these params.